### ISSUE TITLE
User Story 7 - Packing list Delete

### ISSUE THREAD

**Issue by DaveKrauss** [miriam-ertl/my-trips#12]
## Value proposition

**As a** user
**I want to** be able to delete items from my packing list
**so that** i can easily remove items that i no longer need for my trip

## Description

![Image](https://github.com/miriam-ertl/my-trips/assets/128323709/be8393fc-0e34-41b7-b034-2e1242564430)

## Acceptance criteria

- [ ] In the packing list there is a small `x-button` next to each list item
- [ ] Clicking on the button removes the corresponding item from the packing list

## Tasks
- [ ] Create the feature branch: `feature/packing-list-Delete`
- [ ] Add a delete button to each created list item, that will delete given item from the list

**Comment by ahohnsen** [comment]
Heyho,

I noticed the following:

Your US is about your packing list but you do not mention it once in your Value Proposition. 🤔 I would rephrase it to be a bit more specific:

_As a user,
I want to be able to delete items from my packing list,
So I can easily remove items I no longer need for my trip._ 

Same in your acceptance criteria - I suggest to be a bit more specific:

- [ ] In the packing list there is a small x-button next to each list item
- [ ] Clicking on the button removes the corresponding item from the packing list

**Comment by JessicaLoers** [comment]
**well done✨**
